lib/reply.js

```diff
@@ -573,12 +573,17 @@ function handleError (reply, error, cb) {
         message: { value: error.message || '' },
         statusCode: { value: statusCode }
       }))
+
+    if (typeof payload !== 'string') {
```

Comment:
wouldn't it be better to check this only if `serializerFn !== true` ? Because `serializeError` will always return a proper serialized string isn't it?

Reply:
Right, added